### PR TITLE
Implemented school district intent #15

### PR DESCRIPTION
**Pull request by taratip**
For issue  #15 
Added school district intent including unit test

### CODE REVIEW

**Comment by jmartini** [comment]
Hey @taratip I am still having issues after your latest commits. Do you expect this to be working or should I hold off testing?

**Comment by taratip** [comment]
HI @jmartini I still have issues after last commit. Can you provide some suggestions on how to find the right format for querying?

**Comment by jmartini** [comment]
@taratip two suggestions:

1. You can try to look at the requests made from http://gisweb.brooklinema.gov/whereami/, which is the web app we are basing the project on. Use Chrome's developer tools and look at the network calls for the school district query

2. You can test out queries using the web version of the arcgis server: http://gisweb.brooklinema.gov/arcgis/rest/services/MyGov/GovernmentServices/MapServer/16/query. You don't need to fill out all of those fields, but look at what we are setting in the code and enter them. You should see a valid response, with expected fields, printed at the bottom when you press GET.

File: brooklinevoiceapp/mycity/intents/school_district_intent.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    :return: Output speech string
+    """
+
+    logger.debug("Getting response for address " + str(address))
+    features = get_school_district_json(address)
+    logger.debug("school district response:", features)
+
+    try:
+        first_feature = features[0]
+        logger.debug(first_feature)
```
Comment (Joshhw) [inline brooklinevoiceapp/mycity/intents/school_district_intent.py:60]: could you remove this debug statement.

**Pull request merged**